src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -911,11 +913,19 @@ const Authenticate = (() => {
         }
     };
 
-    const handleComplete = () => {
+    const handleComplete = (data) => {
+        const document_ids = [];
+        Object.keys(data).forEach(key => {
+            document_ids.push(data[key].id);
+        });
```

Comment:
```suggestion
        const document_ids = Object.keys(data).map(key => data[key].id);
```
i think it would be cleaner if we do it like this